.github/dependabot.yml

```diff
@@ -9,6 +9,9 @@ updates:
       - dependencies
     schedule:
       interval: daily
+    ignore:
+      - versions:
+        - "*-SNAPSHOT"
```

Comment:
I don't know if it will work. The [official documentation](https://docs.github.com/en/code-security/dependabot/working-with-dependabot/dependabot-options-reference#ignore--) does not say anything about ignoring version like this.

If it does not work, we can try the opposite approach - use `allow` [statement](https://docs.github.com/en/code-security/dependabot/working-with-dependabot/dependabot-options-reference#allow--) instead:

```yaml
    allow:
      - dependency-name: '*'
        dependency-type: production
```

Reply:
BTW: you can try to test it locally using [dependabot-cli](https://github.com/dependabot/cli)